downloader/src/main/kotlin/Downloader.kt

```diff
@@ -66,7 +68,7 @@ class Downloader(private val config: DownloaderConfiguration) {
      * Download the source code of the [package][pkg] to the [outputDirectory]. A [Provenance] is returned on success or
      * a [DownloadException] is thrown in case of failure.
      */
-    fun download(pkg: Package, outputDirectory: File): Provenance {
+    fun download(pkg: Package, outputDirectory: File, dryRun: Boolean = false): Provenance {
```

Comment:
Please explain the `dryRun` argument in the function docs.

Reply:
Done for all public functions.